src/Network/Network/ChangeLog.md

```diff
@@ -19,26 +19,38 @@
 --->
 
 ## Upcoming Release
-
-## Version 7.9.0
-* Onboarded `Microsoft.VideoIndexer/accounts` to private link cmdlets
-* Added support to create, get and delete Bastion shareable links
-    - `New-AzBastionShareableLink`
-    - `Get-AzBastionShareableLink`
-    - `Remove-AzBastionShareableLink`
 * Fixed a bug in cmdlet `Invoke-AzFirewallPacketCapture` which caused the packet capture request to be stuck in a waiting for activation state. 
 * Updated cmdlet to add the property of `Sensitivity`, and updated corresponding cmdlets.
     - `New-AzApplicationGatewayFirewallPolicyManagedRuleOverride`
 * Added support for `DefaultOutboundAccess` property in `Set-AzVirtualNetworkSubnetConfig` command
 * Added support for `EnabledFilteringCriteria` property in `New-AzNetworkWatcherFlowLog` and `Set-AzNetworkWatcherFlowLog` commands
 * Added support of `UserAssignedIdentityId` Property in `New-AzNetworkWatcherFlowLog` and `Set-AzNetworkWatcherFlowLog` commands
-* Added support of `DestinationIPAddress` property in `New-AzPrivateLinkService` command
```

Comment:
removals in this file look incorrect, everything that was under 'upcoming' should get recorded in a new version